pkg/gds/secrets/token.go

```diff
@@ -8,15 +8,15 @@ import (
 
 var (
 	chars        = []rune("ABCDEFGHIJKLMNPQRSTUVWXYZabcdefghjkmnpqrstuvwxyz1234567890")
-	specialChars = []rune("#$%&()*+-<=>?@[]^_{|}~")
+	specialChars = []rune("#%&()*+-<=>?@[]^_{}")
```

Comment:
The `$` and `|` characters require escape in bash so I removed them for my sanity